_includes/commands/table-cmds.md

```diff
@@ -0,0 +1,42 @@
+
+| Command                 | Description                                                                |
+| ----------------------- | -------------------------------------------------------------------------- |
+| mceTableSplitCells      | Splits the current merge table cell.                                       |
+| mceTableMergeCells      | Merges the selected cells or opens a dialog if there is no cell selection. |
+| mceTableInsertRowBefore | Inserts a row before the current row.                                      |
+| mceTableInsertRowAfter  | Inserts a row after the current row.                                       |
+| mceTableInsertColBefore | Inserts a column before the current column.                                |
+| mceTableInsertColAfter  | Inserts a column after the current column.                                 |
+| mceTableDeleteCol       | Deletes the current column.                                                |
+| mceTableDeleteRow       | Deletes the current row.                                                   |
+| mceTableCutRow          | Cuts the current row into table clipboard.                                 |
+| mceTableCopyRow         | Copies the current row into table clipboard.                               |
+| mceTablePasteRowBefore  | Paste the clipboard row before the current row.                            |
+| mceTablePasteRowAfter   | Paste the clipboard row after the current row.                             |
+| mceTableDelete          | Deletes the current table.                                                 |
+| mceInsertTable          | Opens the insert table dialog.                                             |
```

Comment:
Again for consistency, should this be `insert/edit`?

Reply:
fixed